abcFinance/ledger.py

```diff
@@ -139,24 +139,22 @@ def book(self, debit, credit, text=""):
             assert value >= 0
             account = self.accounts[name]
             account.add_debit(value)
+            side, _ = account.get_balance()
             if name in self.asset_accounts:
-                side, _ = account.get_balance()
-                assert side == AccountSide.DEBIT
+                assert side != AccountSide.CREDIT, _
```

Comment:
I'd suggest assert side != AccountSide.CREDIT, (side, _)
as side could be balanced or DEBIT

Reply:
@DavoudTaghawiNejad updated